.github/pull.yml

```
@@ -0,0 +1,14 @@
+version: "1"
+rules:
+  - base: master
+    upstream: UncleGoogle:master
+    mergeMethod: none  # Optional, one of [none, merge, squash, rebase, hardreset], Default: none.
+    mergeUnstable: false    # Optional, merge pull request even when the mergeable_state is not clean. Default: false
+    # assignees:              # Optional
```

I'm not sure about practical difference reviewer vs assignees. 

Probably assignee should be a person responsible for the synchronizing between reviewers/ take final decisions and merge?

---

That seems like a reasonable way to break apart those two roles, the assignee owns the work of making it across the finish line.

---

> Probably assignee should be a person responsible for the synchronizing between reviewers/ take final decisions and merge?

Yeah and should probably be you as you know your way around the repo better than anyone else. 
```suggestion
    assignees:              # Optional
      - UncleGoogle
```

---

In this particular case yes, but can't add assign repo owner as a responsible person to make it merge. Firstly because external people won't have permissions to merge, secondly (I'm less confident with this point because we already have required reviewers) code change owner should not merge its own PRs.

IMO assignee can be someone from org members who knows the plugin; but still - each member can merge when we have full approvals anyway.

An idea: assignee can perform a basic manual test for already approved PR to just check it works, before we will have automated functional tests

---

Oh actually that makes a lot more sense than what I thought. I think leaving assignees blank is probably the best for now.